components/consumers/dependency-track/task.yaml

```diff
@@ -21,12 +22,15 @@ spec:
     - name: consumer-dependency-track-owner-annotation
       type: string
       default: "Owner"
+    - name: consumer-dependency-track-debug
+      type: string
+      default: "false"
   workspaces:
     - name: output
       description: The workspace containing the source-code to scan.
   steps:
   - name: run-consumer
-    imagePullPolicy: IfNotPresent
+    imagePullPolicy: Always
```

Comment:
the imagePullPolicy set to Always is only for debugging locally, it should not be set by default